pkgs/applications/networking/cluster/terraform/providers/providers.txt

```diff
@@ -0,0 +1,13 @@
+# lines starting with a # are comments
+
+# The accepted format (double escape all grep expresssions):
+#  <organisation>         [grep-expression] [grep-v-expression]  - include all repositories in the organisation.
+#                                                                   grep-expression:   filter repo matching the expression
+#                                                                   grep-v-expression: filter repo not matching the expression
+#  <organisation>/<repo>                                         - include only the named repository.
+
+# include all terraform-providers
+terraform-providers terraform-provider- terraform-provider-azure-classic\\|terraform-provider-scaffolding
```

Comment:
Looks good, but you also need to filter out: terraform-provider-google-beta

```
*** terraform-provider-google-beta ***
fetching https://api.github.com/repos/terraform-providers/terraform-provider-google-beta/git/refs/tags?per_page=100
##O#- #
curl: (22) The requested URL returned error: 404 Not Found
The curl command has failed. Attempt 1/30
```

Reply:
@Mic92 PTAL.